container/docker/engine.py

```diff
@@ -768,6 +775,9 @@ def bootstrap_env(self, temp_dir, behavior, operation, compose_option,
                              builder_img_id=builder_img_id,
                              config=config_yaml,
                              env=os.environ,
+                             playbook=self.playbook,
+                             with_volumes=with_volumes,
```

Comment:
Yes, it is in params. Breaking it out of params and passing in an explicit value makes the template less icky.